TestControllers/TestController.psm1

```diff
@@ -788,6 +788,10 @@ Class TestController
 					-ARMImageName $CurrentTestData.SetupConfig.ARMImageName -OsVHD $global:BaseOsVHD -BuildURL $env:BUILD_URL -TableName $dataTableName
 
 				Upload-TestResultToDatabase -SQLQuery $SQLQuery
+
+				if ($dataTableName -eq "LISAv2Results") {
+					Invoke-IngestKustoFromTSQL -SQLString $SQLQuery
```

Comment:
What if the user does not provide Kusto info? The function returns errors. For those who do not use Kusto (yet), the error should be silent?